src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separateParams = sourceString.split(';');
+  const cssStyles = {};
+
+  separateParams.forEach((el) => {
+    const secondsSplit = el.split(':');
+
+    if (secondsSplit[0] && secondsSplit[1]) {
+      cssStyles[secondsSplit[0].trim()] = secondsSplit[1].trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. The `forEach` loop could be replaced with a combination of `map` and `reduce` to create the styles object in a more declarative way.